ui/chart/src/ratingHistory.ts

```diff
@@ -66,6 +67,21 @@ const styles: ChartPerf[] = [
 
 const oneDay = 24 * 60 * 60 * 1000;
 
+const dateFormat = memoize(() =>
+  window.Intl && Intl.DateTimeFormat
+    ? new Intl.DateTimeFormat(
+        document.documentElement.lang.startsWith('ar-') ? 'ar-ly' : document.documentElement.lang,
+        {
+          month: 'short',
+          day: '2-digit',
+          year: 'numeric',
+        },
+      ).format
+    : (d: Date) => d.toLocaleDateString(),
+);
```

Comment:
Unfortunate code-dupe from `timago.ts` due to the different opts and functions. I gave up on trying to find a way to unify.